docs/installation/local.md

```diff
@@ -68,7 +68,7 @@ CREATE DATABASE oevent WITH OWNER john;
 * Once database is created, exit the psql shell with `\q` followed by ENTER.
 
 
-* **Step 3** - Install bower and frontend requirements. For this you need to be present in the root directory of the project. The root directory contains the file ```bower.json```. When you write ```bower install```, it finds bower.json and installs the libraries on the system.
+* **Step 3** - Install bower and frontend requirements. For this you need to be present in the root directory of the project. The root directory contains the file ```bower.json```. When you write ```bower install```, it finds bower.json and installs the libraries on the system. Please note that this `bower.json` file is not present in the `nextgen` branch, hence the installation must take place from the development branch only.
 
```

Comment:
@mish24 Thank you for your contribution :smile: 
Instead of the above, please change this to the following:
```
 Bower has been removed as a dependency in the nextgen version of the project. Please skip this step if working on nextgen branch
```